utils/auth.py

```
@@ -0,0 +1,69 @@
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="/users/login")
+# oauth2 인증 스킴/ Bearer<token> 형식, users/login에서 발급받는다
+
+def hash_password(password: str) -> str:
+    print("[hash_password] type:", type(password))
+    try:
+        print("[hash_password] len(chars):", len(password))
+        print("[hash_password] len(bytes):", len(password.encode("utf-8")))
+        print("[hash_password] preview:", password[:120])
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)